.github/workflows/cypress_workflow.yml

```diff
@@ -42,18 +40,65 @@ env:
   COMMENT_TAG: '[MANUAL CYPRESS TEST RUN RESULTS]'
   COMMENT_SUCCESS_MSG: ':white_check_mark: Cypress test run succeeded!'
   COMMENT_FAILURE_MSG: ':x: Cypress test run failed!'
+  LATEST_VERSION: '2.17.0'
 
 jobs:
   cypress-tests:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-          group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        include:
```

Comment:
Will this work and help organize?
```
matrix:
  config: [standard, query_enhanced, dashboard]
  group: [1, 2, 3, 4, 5, 6, 7, 8, 9]
  include:
    - config: query_enhanced
      group: 10
      test_location: ftr
    - config: dashboard
      group: 11
      test_location: source
  exclude:
    - config: query_enhanced
      group: [1,2,3,4,5,6,7,8,9]
    - config: dashboard
      group: [1,2,3,4,5,6,7,8,9]
```

Reply:
+1,  using matrix to create the right config could make it shorter. but i do see a benefit of making it this granular as it we have more control. as it leave spaces in the future where we migrate the tests from ftr to this repo